src/components/organisms/NoteDetail.tsx

```diff
@@ -465,6 +579,28 @@ class NoteDetail extends React.Component<NoteDetailProps, NoteDetailState> {
 
     return (
       <Container>
+        {viewMode !== 'preview' &&
+          this.state.showSearch &&
+          this.codeMirror != null && (
+            <SearchBarContainer
```

Comment:
How do you think if we move the usage of LocalSearch component to CodeEditor component? The component only visible when code editor is ready. I think we can add a property, `enabledLocalSearch`, so adopters can decide to use this feature or not with this flag.

Reply:
I am not sure where exactly to put it, the CodeEditor is a CustomizedCodeEditor, you mean above that? Or inside that component? (see below comment for more details)